src/constants/version-path.ts

```diff
@@ -3,3 +3,9 @@ export const VERSION_IN_PATH_REGEX = /v\d+\.\d+\.(\d+|\w+)/i
 
 /** This REGEX is used to parse a Terraform Enterprise version from a URL */
 export const TFE_VERSION_IN_PATH_REGEXP = /v[0-9]{6}-\d+/i
+
+/**
+ * TODO - validate the expected versioned integrations paths for other products
+ * Left the 'v' as optional here in case there is variation between the products
+ */
+export const VERSION_IN_INTEGRATIONS_PATH_REGEX = /v?\d+\.\d+\.(\d+|\w+)/i
```

Comment:
@BrandonRomano, all of the other docs versioned paths have `v` in the path itself. Is this something we want to align for consistency across the platform with integrations paths? 

If not, we'll have to treat these versioned paths as a one-off, which is okay but I want to make sure we made an intentional choice to remove it.

Reply:
Ah, we should probably include the v in the path, then!